discord-groups/script.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+import { CANNOT_CONTAIN_GROUP, NO_SPACES_ALLOWED } from './constants.js';
+import {
+  getMembers,
+  getDiscordGroups,
+  addGroupRoleToMember,
+  createDiscordGroupRole,
+  getUserSelf,
+} from './utils.js';
+const groupTabs = document.querySelector('.groups-tab');
+const tabs = document.querySelectorAll('.groups-tab div');
+const sections = document.querySelectorAll('.manage-groups, .create-group');
+const loader = document.querySelector('.loader');
+const userIsNotVerifiedText = document.querySelector('.not-verified-tag');
+
+const membersIdNameObject = {};
+const membersData = await getMembers();
+
+// CREATED A MAP OF {id:username}
+membersData.forEach((member) => {
+  if (member.username) membersIdNameObject[member.id] = member.username;
+});
+
+/**
+ * GET SELF DATA
+ */
+const userSelfData = await getUserSelf();
+const userIsVerified = !!userSelfData.discordId;
```

Comment:
It would be more intuitive to start the variable name with `is`
```suggestion
const IsUserVerified = !!userSelfData.discordId;
```